src/main/java/teammates/common/datatransfer/FeedbackResponseAttributes.java

```diff
@@ -162,7 +162,6 @@ public String getJsonString() {
     
     @Override
     public void sanitizeForSaving() {
-        this.feedbackSessionName = Sanitizer.sanitizeTitle(feedbackSessionName);
```

Comment:
So, what is the policy here? Sanitize only the new attributes? In that case, why is course Id sanitized?